moto/route53/models.py

```diff
@@ -1,26 +1,40 @@
+"""
+Route53 models
+"""
 from __future__ import unicode_literals
 
 from collections import defaultdict
 
 import string
 import random
 import uuid
-from jinja2 import Template
-
+from moto.core.utils import unix_time
 from moto.core import BaseBackend, BaseModel
 
+from jinja2 import Template
 
 ROUTE53_ID_CHOICE = string.ascii_uppercase + string.digits
 
 
 def create_route53_zone_id():
+    """
+    Create route53 zone_id
+    :return:
```

Comment:
If your going to add in all the docstrings can you fill out what it returns, parameters etc... [sphinx docstring format](https://thomas-cokelaer.info/tutorials/sphinx/docstring_python.html#template.MainClass1)